### DISCUSSION TITLE
E-commerce with Tina

### DISCUSSION THREAD

**Discussion by TiagoDeSousa** [tinacms/tinacms#2573] [General]
Is Tina suitable to build an e-commerce website?

**Comment by perkinsjr** [comment]
Absolutely! Tina can power any kind of site you might need marketing, e-commerce, blogs, documentation etc. 

Here is an example with Tina and snipcart using markdown files 

https://github.com/perkinsjr/tina-next-snipcart

**Comment by revskill10** [comment]
@perkinsjr I'm not sure about that.

But the real requirements is:

I have inventory service, so when i update the available_inventory of a product, how can i update the changes to tinacms site ?
